OpenRA.Mods.Common/Installer/SourceResolvers/RegistryDirectorySourceResolver.cs

```diff
@@ -33,6 +34,9 @@ public string FindSourcePath(ModContent.ModSource source)
 				if (!(Microsoft.Win32.Registry.GetValue(prefix + source.RegistryKey, source.RegistryValue, null) is string path))
 					continue;
 
+				// Not obvious, but mandatory: This translates DOS paths.
```

Comment:
Can we please change this to something like `// Resolve 8.3 format (DOS-style) paths to the full path.`?